examples/cython/README.md

```diff
@@ -0,0 +1,28 @@
+# Cython in Ray
```

Comment:
It might be nice to include this in the online documentation.

That would require moving this to `doc/source/` and linking to it from `doc/source/index.rst`.

That could be done as a follow up if you prefer.

Reply:
Done!